components/colorloupe/index.css

```diff
@@ -9,87 +9,84 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 .spectrum-ColorLoupe {
-  --spectrum-colorloupe-width: var(--spectrum-color-loupe-width);
-  --spectrum-colorloupe-height: var(--spectrum-color-loupe-height);
+	--spectrum-colorloupe-width: var(--spectrum-color-loupe-width);
+	--spectrum-colorloupe-height: var(--spectrum-color-loupe-height);
 
-  --spectrum-colorloupe-offset: var(--spectrum-color-loupe-bottom-to-color-handle);
-  --spectrum-colorloupe-animation-distance: 8px; /* TODO: replace with forthcoming animation token */
+	--spectrum-colorloupe-offset: var(--spectrum-color-loupe-bottom-to-color-handle);
+	--spectrum-colorloupe-animation-distance: 8px; /* TODO: replace with forthcoming animation token */
 
-  --spectrum-colorloupe-drop-shadow-x: var(--spectrum-drop-shadow-x);
-  --spectrum-colorloupe-drop-shadow-y: var(--spectrum-color-loupe-drop-shadow-y);
-  --spectrum-colorloupe-drop-shadow-blur: var(--spectrum-color-loupe-drop-shadow-blur);
-  --spectrum-colorloupe-drop-shadow-color: var(--spectrum-color-loupe-drop-shadow-color);
+	--spectrum-colorloupe-drop-shadow-x: var(--spectrum-drop-shadow-elevated-x);
+	--spectrum-colorloupe-drop-shadow-y: var(--spectrum-drop-shadow-elevated-y);
+	--spectrum-colorloupe-drop-shadow-blur: var(--spectrum-drop-shadow-elevated-blur);
+	--spectrum-colorloupe-drop-shadow-color: var(--spectrum-drop-shadow-elevated-color);
 
-  --spectrum-colorloupe-outer-border-width: var(--spectrum-color-loupe-outer-border-width);
-  --spectrum-colorloupe-inner-border-width: var(--spectrum-color-loupe-inner-border-width);
-  --spectrum-colorloupe-outer-border-color: var(--spectrum-color-loupe-outer-border);
-  --spectrum-colorloupe-inner-border-color: var(--spectrum-color-loupe-inner-border);
+	--spectrum-colorloupe-outer-border-width: var(--spectrum-color-loupe-outer-border-width);
+	--spectrum-colorloupe-inner-border-width: var(--spectrum-color-loupe-inner-border-width);
```

Comment:
The `color-loupe-outer-border-width` token is set to `border-width-200`, which is 2px. `color-loupe-inner-border-width` is just set to 1px. But there is a `border-width-100` that's 1px I think. I wonder why we didn't use that token.

This is in the tokens/dist. It's been a while since I've been in some of these files, so maybe I'm looking at things wrong. 
```
--spectrum-color-loupe-outer-border-width: var(--spectrum-border-width-200);
--spectrum-color-loupe-inner-border-width:1px;
```

Nothing's wrong with your code, I was just curious!

Reply:
Yeah, I'm seeing that too! Looks like it is indeed coming from the tokens: 
![image](https://github.com/user-attachments/assets/1edc0724-48db-4bd3-ad23-6872716a8e4e)